models/core/staging/core__stg_claims_eligibility.sql

```diff
@@ -12,8 +12,13 @@
 
 
 select
-        cast(member_id as {{ dbt.type_string() }} ) || '-' || cast(enrollment_start_date as {{ dbt.type_string() }} ) || '-' || cast(enrollment_end_date as {{ dbt.type_string() }} )
+        {% if target.type == 'fabric' %}
```

Comment:
Same note about the concat macro.

Reply:
@sarah-tuva
It -can- but still needs conditional logic anyway for the "plan" so I just left it.